backend/apps/beaver_api/beaver_api/code_api/models.py

```diff
@@ -24,6 +28,9 @@ class CodeDocument(UUIDMixin, TimeStampMixin):  # type: ignore
     tags: models.ManyToManyField = models.ManyToManyField(
         Tag, related_name="code_documents"
     )
+    code_content_hash: models.CharField = models.CharField(
```

Comment:
I would remove the default "", you are declaring that it cannot be null and make it similar to null by default.